management_interface/management_interface/forms.py

```diff
@@ -50,12 +50,21 @@ def clean(self):
         self.cleaned_data["nhs_number"] = "".join(
             self.cleaned_data["nhs_number"].split()
         )
+        self.cleaned_data["nhs_number_hash"] = self._generate_nhs_number_hash()
+        already_existing_care_recipient = CareRecipient.objects.filter(
+            nhs_number_hash=self.cleaned_data["nhs_number_hash"]
+        )
+        if already_existing_care_recipient:
+            raise ValidationError(
+                f"This care recipient already exists in the database with provider reference ID "
+                f"{already_existing_care_recipient[0].provider_reference_id}"
```

Comment:
```suggestion
                f"{already_existing_care_recipient[0].provider_reference_id})"
```

Reply:
I assume you wand `(` here as well?

Reply:
The `(` is the in the change suggested above

Reply:
So no need for an extra one :-)